src/Spectre.Cli/Annotations/CommandOptionAttribute.cs

```diff
@@ -49,6 +51,7 @@ public CommandOptionAttribute(string template)
             LongNames = result.LongNames;
             ShortNames = result.ShortNames;
             ValueName = result.Value;
+            IsOptionalValue = result.OptionalValue;
```

Comment:
Rename to `HasOptionalValue`/`ValueIsOptional`?

Reply:
I like `ValueIsOptional` of those two choices